tests/test.rs

```diff
@@ -10,8 +10,8 @@ enum Response {
 }
 
 #[get("/{a}")]
-async fn route(Path((a,)): Path<(String,)>) -> Response {
-    match a.as_ref() {
+async fn route(path: Path<String>) -> Response {
+    match &path.as_ref()[..] {
```

Comment:
Does `path.as_ref().as_str()` work?

Reply:
Or even `into_inner`?

Reply:
#nitpick

Reply:
`as_str` did the tricky and looks cooler